tmt/steps/provision/__init__.py

```diff
@@ -362,7 +362,9 @@ def _export_environment(self, execute_environment=None):
     def ansible(self, playbook):
         """ Prepare guest using ansible playbook """
         playbook = self._ansible_playbook_path(playbook)
+        environment = self._export_environment()
         stdout, stderr = self.run(
+            f'{environment}'
```

Comment:
```suggestion
            f'{self._export_environment()}'
```
Let's use the method directly.